manager/state/raft/storage/wrapper.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package docker.swarmkit.v1;
```

Comment:
Ah ok - as it wasn't a public protocol (like API level, just an implementation detail) I wasn't sure whether it should put it in the public APIs.